cmd/main.go

```diff
@@ -198,6 +191,13 @@ func main() {
 			}
 		}
 
+		// Metadata.
+		metadata, err := server.NewMetadata(
```

Comment:
why is the change of this code section position?

Reply:
Was trying to separate out the arguments for mixer vs recon services in depoyment.yaml template. schema_path is not needed for recon but without it, recon containers were throwing errors. Moved metadata declaration to mixer only, also closer to where it is first used.

Reply:
Got it